### DISCUSSION TITLE
Where `--no-user-install` option may come from?

### DISCUSSION THREAD

**Discussion by hurricup** [ruby/rubygems#5834] [Q&A] [answered]
I could not google for some reason the way to configure rubygems defaults.

For some reason `gem env` contains
```
- :install => "--no-user-install"
```

And I don't know where could it came from.

It is windows chocolatey installation.

**Comment by deivid-rodriguez** [comment, marked as answer]
Hei! Sorry we never answered here. That probably comes from a `.gemrc` file?
